### ISSUE TITLE
About the Reconstructed Voxels

### ISSUE THREAD

**Issue by DuanchuWang** [chaytonmin/Voxel-MAE#36]
```
 self.deconv1 = nn.Sequential(
         nn.ConvTranspose3d(128, 32, 3, padding=1, output_padding=1, stride=2, bias=False),
         nn.BatchNorm3d(32),
         nn.ReLU()
         )
self.deconv2 = nn.Sequential(
         nn.ConvTranspose3d(32, 8, 3, padding=1, output_padding=1, stride=(4,2,2), bias=False),
         nn.BatchNorm3d(8),
         nn.ReLU()
         )
self.deconv3 = nn.Sequential(
        nn.ConvTranspose3d(8, 1, 3, padding=1, output_padding=1, stride=(3,2,2), bias=False),
        )
```

I don't quite understand why we need to deconvolve three times QAQ, so it seems that the final spatial shape of the result does not match the spatial shape of the label.
